views/index.html

```diff
@@ -187,11 +187,13 @@ <h2 class="call-to-action">Let's Talk!</h2>
 
   <!-- This needs an additional class of dark-bg when 'What our clients say' gets added -->
   <div class="copyright-block">
+    <p>dwyl</p>
+    <p>16 Palmers Road</p>
+    <p>London</p>
+    <p>E2 0SY</p>
+    <p>United Kingdom</p>
     <p>
-      Copyright 2016 &copy; dwyl ltd, company number 09525434, registered in England &amp; Wales. All rights reserved.
-    </p>
-    <p>
-      dwyl, 16 Palmers Road, London, E2 0SY, United Kingdom.
+      Copyright 2017 &copy; DWYL LTD incorporated and registered in England and Wales with company number 09525434 whose registered office is at DWYL LTD, 104 Mary Datchelor Close, London, SE5 7DY, United Kingdom.
```

Comment:
Temporarily as discussed in: https://github.com/dwyl/dwyl-site/issues/203#issuecomment-280371446